pom.xml

```diff
@@ -19,6 +19,7 @@
         <guice.version>4.1.0</guice.version>
         <hamcrest.version>1.3</hamcrest.version>
         <mainClass>uk.gov.pay.products.ProductsApplication</mainClass>
+        <powermock.version>1.7.3</powermock.version>
```

Comment:
I would still question the use of powermock/static mocking. This is usually discouraged and there is normally an alternative. We have been using `randomUuid()` elsewhere (adminusers) heavily and we never felt the need to have powermock for testing there. Mockito was good enough

Reply:
That's true. On the other hand `adminusers` is not using the complex `TransactionFlow` process.

Reply:
Disagree. TransactionFlow has nothing to do with the usage of static contexts.